docs/src/Video.doc.js

```diff
@@ -57,6 +57,13 @@ card(
         description:
           'Accessibility label for the unmute button if controls are shown',
       },
+      {
+        name: 'aspectRatio',
```

Comment:
It might be nice to use the same conventions as Image here - `naturalWidth` & `naturalHeight` instead of asking people to do some math. `aspectRatio` on it's own implies a horizontal ratio.

Reply:
I chose to go the route of `LetterBox` which takes in an `aspectRatio`. Looking at `Image` it takes in the width and height separately but still only uses them together in one spot to actually define the aspect ratio. Its confusing because the formula for aspect ratio is width / height (see https://www.extron.com/company/article.aspx?id=aspectratio) but for the padding trick to work you do height / width (hence the 1 / ratio in my code). I think this math is ok for the client to do as long as the docs describe the correct division which i included. I also like how `aspectRatio` forces the existence into one prop, either defined or defaulted, whereas splitting it into separate width and height would require us to validate that both are present (if one is supplied) otherwise the math gets hairy

Reply:
Ok!